client/src/components/Home.tsx

```diff
@@ -1,54 +1,50 @@
 import {
     List,
-    FloatingActionButton,
     ListItem,
     Subheader,
 } from 'material-ui';
 
-import ContentAdd from 'material-ui/svg-icons/content/add';
 import {LessonSummary} from '../reducers/lessonSummaryReducer';
+import {Course} from '../reducers/courseReducer';
 import Page from './util/Page';
 import * as React from 'react';
 
 export interface HomeProps {
+    courses?: Course[];
     lessons?: LessonSummary[];
     loading?: boolean;
     onCreateLessonClick?: () => void;
```

Comment:
Do we still need lessons, LessonSummary, and onCreateLessonClick in the home component?